eng/common/scripts/Package-Properties.ps1

```diff
@@ -113,6 +113,11 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
     $diff = Get-Content $InputDiffJson | ConvertFrom-Json
     $targetedFiles = $diff.ChangedFiles
 
+    $customDependentPackagesHandler = $false
+    if ($DependentPackagesFromPackageSetFn -and (Test-Path "Function:$DependentPackagesFromPackageSetFn")) {
```

Comment:
Do we need another function here or can we handle this inside of function where were create the PackageProperties?

Reply:
It looks like you already are in your PR https://github.com/Azure/azure-sdk-for-python/pull/36635/files#diff-1d28dc86baa237ca6efe6224431142a9e8bbff66fa10a2c20c69031a22f5b387R62

Reply:
If you don't implement the function, yes, we use the common one.

Reply:
OK I could see this potentially being useful for mapping changes outside of the sdk folders, like `eng` to cause some set of test packages like template or some core ones. 

That actually makes me think we might also need a fallback so that if no packages are discovered that we use at least one test package.

Reply:
> That actually makes me think we might also need a fallback so that if no packages are discovered that we use at least one test package.

And this functionality will allow us to handle that!